README.md

```diff
@@ -80,6 +80,28 @@ chmod +x tezos-client
 
 Run `./tezos-client` or add it to your PATH to be able to run it anywhere.
 
+## Brew tap for macOS
```

Comment:
Maybe let's link here somewhere around the top of README? Or maybe some kind of a ToC. It's currently hard to understand whether MacOS is supported.

Reply:
Yes, I agree, it's time to add ToC to the README